tests/Microsoft.Identity.Test.Performance/Microsoft.Identity.Test.Performance.csproj

```diff
@@ -2,15 +2,16 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <Configurations>Debug;Release;Debug + MobileApps</Configurations>
+    <TargetFramework>net8.0</TargetFramework>
+    <Configurations>Debug;Release;</Configurations>
     <!-- Uncomment only when running EtwProfiler diagnoser on Release-->
     <!-- https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation#debugtype -->
     <!--<DebugType>pdbonly</DebugType>
     <DebugSymbols>true</DebugSymbols>-->
     <LangVersion>latest</LangVersion>
```

Comment:
>latest [](http://example.com/codeflow?start=17&length=6)

learned from Stephen Toub that you should define the lang version -> see [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2184)

also, do you have a Directory.Build.Props file for shared content across projects?

Reply:
I don't quite understand why `latest` mapped to pre-11? [`latest` is based on the compiler version](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#c-language-version-reference), so should map to 12 in this case? If the `LangVersion` is absent, then the TFM dictates the language version.

Yes, we have three Directory.Build.props: https://github.com/search?q=repo%3AAzureAD%2Fmicrosoft-authentication-library-for-dotnet+path%3A**%2Fdirectory.build.props&type=code